viewer/vueapp/src/components/settings/Settings.vue

```diff
@@ -2499,11 +2499,11 @@ export default {
 
           if (updateTheme) {
             let now = Date.now();
-            if ($('link[href^="user.css"]').length) {
-              $('link[href^="user.css"]').remove();
+            if ($('link[href^="api/user/css"]').length) {
```

Comment:
still hurts deeply that it isn't api/user.css, at least we support both

Reply:
💔